how-to/workspace-platform-starter/client/src/framework/launch.ts

```diff
@@ -377,8 +385,39 @@ async function launchView(viewApp: PlatformApp): Promise<PlatformAppIdentifier |
 	if (!viewExists) {
 		try {
 			const platform = getCurrentSync();
-			const createdView = await platform.createView(manifest);
-			identity = createdView.identity;
+
+			if (!isEmpty(viewApp.launchPreference?.bounds) || !isEmpty(viewApp?.launchPreference?.options?.view)) {
+				await platform.createWindow({
+					workspacePlatform: !isEmpty(viewApp.launchPreference?.options?.view?.hostUrl)
```

Comment:
Shouldn't we look up the identity of the view, by getting the layout of the created window and return it, like the else case does ?

Reply:
done with extra checks to ensure it is the view we expected.